src/Persistence/ResetPasswordRequestRepositoryInterface.php

```diff
@@ -21,4 +21,9 @@ public function findResetPasswordRequest(string $selector): ?ResetPasswordReques
     public function getMostRecentNonExpiredRequestDate(object $user): ?\DateTimeInterface;
 
     public function removeResetPasswordRequest(ResetPasswordRequestInterface $resetPasswordRequest): void;
+
+    /**
+     * Remove all expired reset password request objects from persistence
```

Comment:
Needs an `@return` to describe the meaning of the `int` return type

Reply:
done